hasjob/templates/detail.html

```diff
@@ -430,20 +426,67 @@
       });
       return false;
     });
-  });
-  $(function () {
+    // section
     var triggered_related = false;
+    var triggered_viewstats = false;
     // Fetch the related posts once the given view is on the display.
     // triggered_related is a boolean that makes sure we dont call the
     // related posts endpoint more than once.
     $('#apply-info').appear().on('appear', function(event, element) {
-      if(!triggered_related) {
+      if (!triggered_related) {
         triggered_related = true;
         $.get(window.location.pathname + '/related', function (data) {
-            $('#stickie-area').html(data);
+            $('#related-posts').html(data);
           }
         );
       }
+
+      {%- if is_siteadmin or post.admin_is(g.user) or (g.user and g.user.flags.is_employer_month) %}
+      if (!triggered_viewstats) {
+        triggered_viewstats = true;
+        var UNITTYPE = {
+          'q': '15m',
+          'h': 'hour',
+          'd': 'day'
+        }
+        $.get(window.location.pathname + '/viewstats', function(data) {
```

Comment:
Don't use `window.location.pathname`. This will break when we move to continuous scrolling posts (#271). Instead insert the URL from Jinja2.